pkgs/build-support/ocaml/dune.nix

```diff
@@ -39,7 +47,7 @@ stdenv.mkDerivation ({
 
   strictDeps = true;
 
-} // (builtins.removeAttrs args [ "minimalOCamlVersion"  "duneVersion" ]) // {
+} // (builtins.removeAttrs args [ "minimalOCamlVersion" "useDune2" "duneVersion" ]) // {
```

Comment:
When I replaced `useDune2 = true` with `duneVersion = "2"` it caused a rebuild so I remove so other project that depend on nixpkgs might not need a rebuild.

Reply:
This is unnecessary.